### ISSUE TITLE
Axis Discrepancy between Pytorch and Tensorflow

### ISSUE THREAD

**Issue by abhimanyuchadha96** [ultralytics/yolov5#7708]
https://github.com/ultralytics/yolov5/blob/c4cb7c684ad6baf4081eb59ee1c27aa97bed4ebe/models/tf.py#L161
https://github.com/ultralytics/yolov5/blob/c4cb7c684ad6baf4081eb59ee1c27aa97bed4ebe/models/common.py#L124

The BottleNeckCSP layer class for Pytorch and Tensorflow have different axis parameters, why is that?
In Pytorch it is concatenated over the first dimension whereas in tensorflow it is done over the third dimension.

**Comment by glenn-jocher** [comment]
@abhimanyuchadha96 TF uses a different ordering. In PyTorch the order is BCWH.

**Comment by abhimanyuchadha96** [comment]
Ok

**Comment by glenn-jocher** [comment]
@abhimanyuchadha96 feel free to refer to the Ultralytics documentation for more details about how YOLOv5 processes data and the underlying architecture. If you have any more questions, feel free to ask!
